doc/Publishing.md

```diff
@@ -11,6 +12,32 @@ In order to release one should:
 - tag the published version
 - announce release is done
 
+## Updating the release page on Eclipse Foundation
+
+`One week` before the release, the project's release page should be updated in order to announce upcoming changes.
+- Login to eclipse: https://projects.eclipse.org/projects/ecd.theia
+- On the right side panel, select Release / Create a new release
+
+    Create a new version
+    - Name: enter the new version for this release. (ex: 1.2.0).
+    - Date: enter the date for this release.
+
+    Select the edit tab
+    - In "The Basic" section
+        - Select the  "Type A" in the IP Due Diligence type.
+    - In the "Project Plan"
+        - Deliverables section
+            - Add the following line pointing to the modifications where the end-user can have a look.
+
+            `[Changelog](https://github.com/eclipse-theia/theia/blob/master/CHANGELOG.md)`
```

Comment:
I do not like using a link here. Please paste the content  from CHANGELOG. It needs to be done is 2 parts. This one and just below there's also "Compatibility' that should be filled-in with breaking changes from CHANGELOG - see release 1.0 as example.

Reply:
For compatibility, start with "Breaking changes:"

Both these sections should be updated following the release so we have the final content.

Reply:
Added a section pre-release and a section post-release. Removed the link.